lib/kubeclient.rb

```diff
@@ -47,6 +47,8 @@ def get_resource_name(entity)
       end
     end
 
+    public
```

Comment:
this fixes the get_/delete_/etc. visibility in ruby 2.1 (define_method scope is protected (2.1) vs public (2.0)... so I had to move this back here)